x-pack/plugins/fleet/public/applications/fleet/sections/settings/components/edit_output_flyout/index.tsx

```diff
@@ -118,174 +114,17 @@ export const EditOutputFlyout: React.FunctionComponent<EditOutputFlyoutProps> =
-            'xpack.fleet.settings.editOutputFlyout.sslCertificateAuthoritiesInputPlaceholder',
-            {
-              defaultMessage: 'Specify certificate authority',
-            }
-          )}
-          label={i18n.translate(
-            'xpack.fleet.settings.editOutputFlyout.sslCertificateAuthoritiesInputLabel',
-            {
-              defaultMessage: 'Server SSL certificate authorities (optional)',
-            }
-          )}
-          multiline={true}
-          sortable={false}
-          {...inputs.sslCertificateAuthoritiesInput.props}
-        />
-        <EuiFormRow
-          fullWidth
-          label={
-            <FormattedMessage
-              id="xpack.fleet.settings.editOutputFlyout.sslCertificateInputLabel"
-              defaultMessage="Client SSL certificate"
-            />
-          }
-          {...inputs.sslCertificateInput.formRowProps}
-        >
-          <EuiTextArea
-            fullWidth
-            rows={5}
-            {...inputs.sslCertificateInput.props}
-            placeholder={i18n.translate(
-              'xpack.fleet.settings.editOutputFlyout.sslCertificateInputPlaceholder',
-              {
-                defaultMessage: 'Specify ssl certificate',
-              }
-            )}
-          />
-        </EuiFormRow>
-        {(output && output?.ssl?.key) || !useSecretsStorage ? (
-          <EuiFormRow
-            fullWidth
-            label={
-              <FormattedMessage
-                id="xpack.fleet.settings.editOutputFlyout.sslKeyInputLabel"
-                defaultMessage="Client SSL certificate key"
-              />
-            }
-            {...inputs.sslKeyInput.formRowProps}
-          >
-            <EuiTextArea
-              fullWidth
-              rows={5}
-              {...inputs.sslKeyInput.props}
-              placeholder={i18n.translate(
-                'xpack.fleet.settings.editOutputFlyout.sslKeyInputPlaceholder',
-                {
-                  defaultMessage: 'Specify certificate key',
-                }
-              )}
-            />
-          </EuiFormRow>
-        ) : (
-          <SecretFormRow
-            fullWidth
-            title={i18n.translate('xpack.fleet.settings.editOutputFlyout.sslKeySecretInputTitle', {
-              defaultMessage: 'Client SSL certificate key',
-            })}
-            {...inputs.sslKeySecretInput.formRowProps}
-            onUsePlainText={onUsePlainText}
-            cancelEdit={inputs.sslKeySecretInput.cancelEdit}
-          >
-            <EuiTextArea
-              fullWidth
-              rows={5}
-              {...inputs.sslKeySecretInput.props}
-              placeholder={i18n.translate(
-                'xpack.fleet.settings.editOutputFlyout.sslKeySecretInputPlaceholder',
-                {
-                  defaultMessage: 'Specify certificate key',
-                }
-              )}
-            />
-          </SecretFormRow>
-        )}
-      </>
+      <OutputFormLogstashSection
```

Comment:
this is a refactor to move `logstash` and `elasticsearch` fields to their own `output_form_*` files